gems/pending/openstack/openstack_event_monitor.rb

```diff
@@ -102,6 +99,14 @@ def self.event_monitor_key(options)
     options.values_at(:hostname, :username, :password)
   end
   private_class_method :event_monitor_key
+
+  def amqp_event(_delivery_info, metadata, payload)
+    OpenstackAmqpEvent.new(payload,
```

Comment:
@blomquisg Thanks for comment, "amqp" was replaced with "events" in code which is common for both RabbitMQ and Ceilometer.